src/main/java/nl/nn/testtool/filter/View.java

```diff
@@ -36,86 +40,114 @@
  */
 @Dependent
 public class View implements BeanParent {
-	protected String name;
-	protected String nodeLinkStrategy;
-	private @Setter @Getter @Inject @Autowired LogStorage debugStorage;
-	private @Setter @Getter @Inject @Autowired List<String> metadataNames;
-	private Map<String, String> metadataFilter;
-	private List<CheckpointMatcher> checkpointMatchers;
-	private BeanParent beanParent;
-	private Echo2Application echo2Application;
-
-	protected enum NodeLinkStrategy {
-		PATH,
-		CHECKPOINT_NUMBER,
-		NONE
-	}
-
-	public void setName(String name) {
-		this.name = name;
-	}
-
-	public String getName() {
-		return name;
-	}
-
-	public void setNodeLinkStrategy(String nodeLinkStrategy) {
-		this.nodeLinkStrategy = nodeLinkStrategy;
-	}
-
-	public String getNodeLinkStrategy() {
-		if (nodeLinkStrategy == null) {
-			return NodeLinkStrategy.PATH.toString();
-		}
-		return nodeLinkStrategy;
-	}
-
-	public void setMetadataFilter(Map<String, String> metadataFilter) {
-		this.metadataFilter = metadataFilter;
-	}
-
-	public Map<String, String> getMetadataFilter() {
-		return metadataFilter;
-	}
-
-	public void setCheckpointMatchers(List<CheckpointMatcher> checkpointMatchers) {
-		this.checkpointMatchers = checkpointMatchers;
-	}
-
-	/**
-	 * @see nl.nn.testtool.echo2.Echo2Application#initBean()
-	 */
-	public void initBean(BeanParent beanParent) {
-		this.beanParent = beanParent;
-		this.echo2Application = Echo2Application.getEcho2Application(beanParent, this);
-	}
-
-	public BeanParent getBeanParent() {
-		return beanParent;
-	}
-
-	public Echo2Application getEcho2Application() {
-		return echo2Application;
-	}
-
-	public String isOpenReportAllowed(Object StorageId) {
-		return ReportsComponent.OPEN_REPORT_ALLOWED;
-	}
-
-	public boolean showCheckpoint(Report report, Checkpoint checkpoint) {
-		boolean match = false;
-		if (checkpointMatchers == null) {
-			match = true;
-		}
-		for (int i = 0; !match && i < checkpointMatchers.size(); i++) {
-			CheckpointMatcher checkpointMatcher = checkpointMatchers.get(i);
-			match = checkpointMatcher.match(report, checkpoint);
-		}
-		return match;
-	}
-
-	public String toString() {
-		return getName();
-	}
+    protected String name;
```

Comment:
Can you keep whitespaces on tabs?

Reply:
This is the default java formatting in IntelliJ

Reply:
Please keep the formatting as is (like all other files in the project) to keep the project consistent and make reviewing easier

Reply:
I would like to, however there's not way to enforce this or tell my IDE to use a specific format, as there is none defined. 
The only way to do this that I see is to ask ChatGPT to fix my indentation by supplying the new file and the old file with the indentation. I don't think that's supposed to be the way as it makes contributing to this project very cumbersome. Can you tell me how I can do this more efficiently?

Reply:
I tried to ask ChatGPT and it couldn't even do it.

Reply:
I fixed it as much as possible